src/tilemaker/scripts/run_tilemaker.sh

```diff
@@ -33,21 +33,67 @@ function generate_tiles_from_shapefile() {
 
     # Get bounds. ogrinfo can handle big files, but doesn't provide tons of output flexibility, so
     # parse out the answer from the blob of text it returns.
-    TL_BOUNDS=$(ogrinfo -so -al "/data/${TL_SHAPEFILE_NAME}.shp" \
+    export TL_BOUNDS=$(ogrinfo -so -al "/data/${TL_SHAPEFILE_NAME}.shp" \
         | grep Extent | sed -E 's/.*\((.*), (.*)\) - \((.*), (.*)\).*/\1 \2 \3 \4/')
 
     # Make tiles and upload them to S3.
-    # 'tl' only has totally-quiet or very-verbose (every tile) output options, so this filters
+    # 'tl' only has totally-quiet or very-verbose (every tile) output options, so we filter
     # the output to show the first line then every 1000th line thereafter.
-    /usr/bin/time -f "\nTIMING: %C\nTIMING:\t%E elapsed %Kkb mem\n" \
-    tl copy --min-zoom "${TL_MIN_ZOOM}" --max-zoom "${TL_MAX_ZOOM}" \
-        --bounds "${TL_BOUNDS}" \
-        "mapnik:///opt/pfb/tilemaker/styles/${TL_SHAPEFILE_NAME}_style.xml" \
-        "s3://${AWS_STORAGE_BUCKET_NAME}/${PFB_S3_TILES_PATH}/${TL_SHAPEFILE_NAME}/{z}/{x}/{y}.png" \
-        | awk 'NR == 1 || NR % 1000 == 0'
+    #
+    # The work is done in the two functions, which are spawned in many (relatively) bit-sized
+    # pieces with `parallel`
+
+    # Define these once to save space below. They need to be exported because they'll be used
+    # in the functions.
+    export TL_INPUT="mapnik:///opt/pfb/tilemaker/styles/${TL_SHAPEFILE_NAME}_style.xml"
+    export TL_OUTPUT="s3://${AWS_STORAGE_BUCKET_NAME}/${PFB_S3_TILES_PATH}/${TL_SHAPEFILE_NAME}/{z}/{x}/{y}.png"
```

Comment:
Yeah, I'm not sure how fast each thread will be making tiles in the production environment, but I would hope it's at least a few per second, and if we could end up with 20 jobs running 8 threads each, that's 160 threads, so anything more than 2/second would put us over the 300/s PUT/LIST/DELETE limit.  So a support case seems like a good idea.